internal/utility/fileposition/string.go

```diff
@@ -43,12 +44,12 @@ func ExtractRegexpPositionInBlock(block []string, str string, blockStartLine int
 }
 
 func ExtractDelimitedRegexpPositionInBlock(block []string, str string, blockStartLine int, prefix string, suffix string) *models.FilePosition {
-	// We expect 'str' to be a literal value or in case it is a regexp to be only one capturing group
-	regex := cachedregexp.MustCompile(cachedregexp.QuoteMeta(prefix) + str + cachedregexp.QuoteMeta(suffix))
+	group := fmt.Sprintf("(%s)", str)
```

Comment:
I didn't saw any tests checking if the `str` already contains a group

Reply:
Good catch! Function and tests updated 👍